com.vcvrack.Rack2.yaml

```diff
@@ -0,0 +1,76 @@
+id: com.vcvrack.Rack2
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '23.08'
+sdk: org.freedesktop.Sdk
+command: rack
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --allow=bluetooth
+  - --filesystem=host
+  - --filesystem=xdg-run/pipewire-0
+  - --env=XDG_CURRENT_DESKTOP=GNOME
+
+x-gl-version: &gl-version '1.4'
+x-gl-versions: &gl-versions 23.08;23.08-extra;1.4
+x-gl-merge-dirs: &gl-merge-dirs vulkan/icd.d;glvnd/egl_vendor.d;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+
+add-extensions:
+  org.freedesktop.Platform.GL:
+    directory: lib/GL
+    version: *gl-version
+    versions: *gl-versions
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: *gl-merge-dirs
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+    autoprune-unless: active-gl-driver
+
+  org.freedesktop.Platform.VAAPI.Intel:
+    directory: lib/dri/intel-vaapi-driver
+    version: *runtime-version
+    versions: *runtime-version
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: *gl-merge-dirs
+    download-if: have-intel-gpu
+    enable-if: have-intel-gpu
+    autoprune-unless: have-intel-gpu
+
+modules:
+  - name: rack2free
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/lib/GL
+      - mkdir -p ${FLATPAK_DEST}/dri/intel-vaapi-driver
+      - install -Dm644 metainfo.xml ${FLATPAK_DEST}/share/${FLATPAK_ID}.metainfo.xml
+      - install -Dm755 ${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644  res/icon.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/com.vcvrack.Rack2.png
+      - install -Dm755 rack_wrapper.sh ${FLATPAK_DEST}/bin/rack
+      - cp -a ./ ${FLATPAK_DEST}/
+    sources:
+      - type: file
+        path: com.vcvrack.Rack2.desktop
+      - type: file
+        path: metainfo.xml
+      - type: script
+        dest-filename: rack_wrapper.sh
+        commands:
+          - cd /app
+          - exec /app/Rack --system /app --user $XDG_DATA_HOME $@
+      - type: archive
+        url: https://vcvrack.com/downloads/RackFree-2.5.2-lin-x64.zip
```

Comment:
why not building from source?

Reply:
It's to much for me not being C++ developer. I could build it in standard environment but unlikely in flatpak env.
It's a huge dependencies list: https://github.com/VCVRack/Rack

Reply:
it's not about C++.

Reply:
I would insist on using official distribution of the app as a base.

Reply:
I would insist on building from source.

Reply:
WIP

Reply:
Manifest was reworked to build everything from sources.